db/driver/testdata/seed-test-db.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO plans (type, rate_limit_throughput)
```

Comment:
#PUC small comment at the top on what / how much this seeds.

Reply:
Updated.